packages/p2p/src/components/modal-manager/modals/share-my-ads-modal/share-my-ads-socials/share-my-ads-socials.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import { FacebookShareButton, TelegramShareButton, TwitterShareButton, WhatsappShareButton } from 'react-share';
+import { Icon, Text } from '@deriv/components';
+import { Localize } from 'Components/i18next';
+import classNames from 'classnames';
+
+const ShareMyAdsIcons = () => {
+    // TODO: replace with proper url and message when available
+    const advert_url = window.location.href;
+    const custom_message = 'This is my advert!';
+
+    const share_buttons = [
```

Comment:
Can move this to constants ? 🤔

Reply:
ah yeah i will move these after i merge the next subtask in here, so theres one more task to change the custom message and advert_url but I left these here so i can remember to do that hahaha, but for sure, i'll move these to its own constants file 👍